_rules/css-orientation-lock-media-queries.md

```diff
@@ -0,0 +1,215 @@
+
+- The rule does not take consider existence of any control on the page that can handle or change the orientation on demand.
+
+## Accessibility support
+
+*There are no major accessibility support issues known for this rule.*
+
+## Background
+
+- [Understanding Success Criterion 1.3.4: Orientation](https://www.w3.org/WAI/WCAG21/Understanding/orientation.html)
+- [Managing screen orientation](https://developer.mozilla.org/en-US/docs/Web/API/CSS_Object_Model/Managing_screen_orientation)
+- [MDN Orientation](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/orientation)
+
+## Test Cases
+
+### Passed
+
+#### Passed example 1
+
+A page, where CSS media query styles negate orientation lock, and therefore does not restrict its view or operation.
+
+```html
+<html lang="en">
+  <head>
+    <!--
+      1. The style from the relative stylesheet applies `transform: rotate(90deg)`
+    -->
+    <link rel="stylesheet" href="../test-assets/html-css-lock.css">
+    <!--
+      2. The below style negates the orientation lock
+    -->
+    <style>
+      @media (orientation: portrait) {
+        body {
+          transform: rotate(-90deg);
+        }
+      }
+    </style>
+  </head>
+  <body>
+    <main>
+      Page Content
+    </main>
+  </body>
+</html>
+```
+
+#### Passed example 2
+
+A page (piano application), where orientation lock is essential.
+
+```html
+<html lang="en">
+  <head>
+    <style>
+      @media (orientation: portrait) {
+        body {
+          transform: rotate(90deg);
+        }
+      }
+    </style>
+  </head>
+  <body>
+    <main>
+      <!-- piano application -->
+    </main>
+  </body>
+</html>
+```
+
+### Failed
+
+#### Failed example 1
+
+A page, where orientation is locked by applying `transform` style on `portrait` media query (coming from a relative stylesheet).
+
+```html
+<html lang="en">
+  <head>
+    <!--
+      1. The style from the relative stylesheet applies `transform: rotate(90deg)`
+    -->
+    <link rel="stylesheet" href="../test-assets/html-css-lock.css">
+  </head>
+  <body>
+      Page Content
+  </body>
+</html>
+```
+
+#### Failed example 2
+
+A page, where orientation is locked via `style` specified on the `head`.
+
+```html
+<html lang="en">
+  <head>
+    <!--
+      1. The style from the stylesheet applies `transform`
+    -->
+    <style>
+      @media (orientation: portrait) {
+        body {
+          transform: rotate(-90deg);
+        }
+      }
+    </style>
+  </head>
+  <body>
+      Page Content
+  </body>
+</html>
+```
+
+
+### Inapplicable
+
+#### Inapplicable example 1
+
+A page where there are no styles.
+
+```html
+<html lang="en">
+  <body>
+    I am a page with no styles
+  </body>
+</html>
+```
+
+#### Inapplicable example 2
+
+A page that has no CSS media query styles.
+
+```html
+<html lang="en">
+  <head>
+    <style>
+      html {
+        font-size: 22px;
+      }
+    </style>
+  </head>
+  <body>
+    Page Content
+  </body>
+</html>
+```
+
+#### Inapplicable example 3
```

Comment:
I don't think this one is needed. It is testing the same as example 4 which is thorough